File: Object-Oriented Programming/Códigos/Códigos/POO 4/Animais/Mamifero.php
<?php
require_once 'Animais.php';
class Mamifero extends Animais {
	private $idade;
	private $cor;

	public function getIdade() {
		return $this->idade;
	}

	public function setIdade($idade): void {
		$this->idade = $idade;
	}

	public function getCor() {
		$this->cor = $cor;
	}

	public function setCor($cor): void {
		$this->cor = $cor;
	}

}

?>

File: Programação Orientada a Objetos/Códigos/Códigos/POO 2/Projeto Animais/Mamifero.java
package Proj_Animais;

public class Mamifero extends Animais {
	// Atributos
	public String cor;
	public String nome;
	public int idade;

	public Mamifero() {
		
	}
	
	// Metodos (opcional)
	public void Andar() {
		// Código...
	}

	// Metodos Especiais
	public String getCor() {
		return cor;
	}

	public void setCor(String cor) {
		this.cor = cor;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public int getIdade() {
		return idade;
	}

	public void setIdade(int idade) {
		this.idade = idade;
	}

}
